mod_auth/forms.py

```diff
@@ -5,8 +5,7 @@
 
 from flask_wtf import FlaskForm
 from wtforms import PasswordField, SelectField, StringField, SubmitField
-from wtforms.fields.html5 import EmailField
-from wtforms.fields.simple import PasswordField
+from wtforms.fields import EmailField, PasswordField
```

Comment:
Please revert this change, it's unnecessary. See #616

Reply:
Okay Sir, will remove this now.